### DISCUSSION TITLE
iOS App crash

### DISCUSSION THREAD

**Discussion by Xovval** [community/community#33649] [Mobile]
Steps to reproduce:
- Start the app
- Tap (+) to create a new issue
- Long press on any recommended repository in the appearing screen
- Pull down in order to close the screen again

Doing so, the app crashes.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.81.0 (78683238)
Device: iPhone 13 (iOS 16.0)
</details>

**Comment by eliperkins** [comment]
Thanks for this crash report, @Xovval ! I can reproduce this 100% of the time as well. 🙇 

We've got a stack trace to go on now, so a fix should be on the way in the next beta builds shortly. I'll keep you posted when this bug fix lands.
